public/index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8" />
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
```

Comment:
as a note, we can use bootstrap cdn or install bootstrap and bundle it up with our project. pros and cons to each. there is also a react based bootstrap called `react-bootstrap` for better react-ish integration.

Reply:
Thanks, I'll stick with the CDN for now but good to know it can be bundled